deploy/addons/olm/olm.yaml.tmpl

```diff
@@ -78,9 +78,9 @@ spec:
           command:
           - /bin/olm
           args:
-          - -namespace
+          - --namespace
```

Comment:
what is the reason for the additional -- ?

Reply:
old OLM versions were using `flags` at some point they switched to `github.com/spf13/pflag` library, hence move from `-` to `--`